src/Blankslate/Blankslate.tsx

```diff
@@ -54,18 +23,26 @@ export type BlankslateProps = React.PropsWithChildren<{
   spacious?: boolean
 }>
 
+const StyledBlankslate = styled.div`
+  --borderColor-default-local-fallback: ${get('colors.border.default')};
+  --fgColor-muted-local-fallback: ${get('colors.fg.muted')};
```

Comment:
We don't have the new Primitives colors yet, so we need to provide fallbacks using JS.